vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -193,10 +193,18 @@ export default class SbcHeader extends NavigationMixin {
       this.loadUserInfo()
     })
 
-    if (ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken)) {
+    if (ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken) && (this.accountType !== 'IDIR')) {
       const lastUsedAccount = this.getLastAccountId()
       await this.syncUserSettings(lastUsedAccount)
-      this.persistAndEmitAccountId()
+
+      /*
+     Emit event to downstream only when account id is changed.
+     ie ; on account switch or on when the app is loaded. Otherwise need not to propagate the changes
+     */
+      const isAccountChanged = lastUsedAccount !== this.currentAccount.id.toString()
```

Comment:
suggetion: better to add a null check for the currentAccount

Reply:
sure.Will add it